core/src/components/cat-pagination/cat-pagination.tsx

```diff
@@ -81,7 +86,7 @@ export class CatPagination {
               a11yLabel={i18n.t('pagination.prev')}
               icon={this.iconPrev}
               iconOnly
-              onClick={() => (this.page = this.page - 1)}
+              onClick={() => this.prevPage()}
```

Comment:
as this has not been checked before either, I would just rely on the fact, that the button will be disabled. This just, call setPage here. Same goes for nextPage.